app/models/account.rb

```diff
@@ -55,6 +55,7 @@ class Account < ApplicationRecord
   scope :providers, -> { where(provider: true) }
 
   scope :providers_with_master, -> { where.has { (provider == true) | (master == true) } }
+  scope :providers_without_master, -> { providers.not_master }
```

Comment:
i would say this scope is: `not_master_providers`

Reply:
mmmm... that name sounds more confusing to me 😕 It sounds like it could be "not master and not provider" 🤔

Reply:
ok 👍 It sounds good 👍